[Term Entry] Lua Math .abs() (#3671)
Codecademy/docs: content/lua/concepts/mathematical-library/terms/abs/abs.md

---
Title: 'abs()'

> Description: 'Converts given number to absolute value'

Reviewer: ```suggestion
Description: 'Returns the absolute value of a given number.'
```

Subjects:
  - 'Computer Science'
  - 'Game Development'
Tags:
  - 'Math'
  - 'Functions'
CatalogContent:
  - 'learn-lua'
  - 'paths/computer-science'
---

> The **`abs()`** function in Lua's math library converts a number  to its absolute value.

Reviewer: ```suggestion
The **`abs()`** function in Lua's math library returns the absolute value of a given number.
```

## Syntax

```pseudo
math.abs(number)
```

> - `number`: The number.

Reviewer: ```suggestion
- `number`: The number to be addressed.
```

## Example

> In this example, `math.abs()` converts -5 to 5.

Reviewer: ```suggestion
This example demonstrates a basic implementation of `math.abs()`.
```

Reviewer: Including the output here is redundant
